trinity/utils/logging.py

```diff
@@ -8,30 +9,47 @@
 
 from typing import Tuple, Callable
 
+from trinity.config import (
+    ChainConfig,
+)
 
-def setup_trinity_logging(level: int) -> Tuple[Logger, Queue, handlers.QueueListener]:
+
+def setup_trinity_logging(chain_config: ChainConfig, level: int) -> Tuple[Logger, Queue, handlers.QueueListener]:
     from .mp import ctx
 
-    log_queue = ctx.Queue()
+    log_queue_stream = ctx.Queue()
+    log_queue_file = ctx.Queue()
 
     logging.basicConfig(level=level)
     logger = logging.getLogger('trinity')
 
-    handler = logging.StreamHandler(sys.stdout)
+    log_dir = chain_config.data_dir / "logs"
+    log_file = log_dir / "trinity.log"
```

Comment:
Oh, and this logic should probably move into the `ChainConfig` class itself so that you can just do `ChainConfig.logfile_path` to access the appropriate path.